backend/video/src/subtitle.py

```diff
@@ -7,12 +7,14 @@
 
 import json
 import os
+import re
 from datetime import datetime
+from operator import itemgetter
 
 import requests
 from common.src.env_settings import EnvironmentSettings
 from common.src.es_connect import ElasticWrap
-from common.src.helper import requests_headers
+from common.src.helper import orderedSet_from_options, requests_headers
```

Comment:
You can import that directly from yt-dlp, so we don't need to duplicate that here:
```python
from yt_dlp.utils import orderedSet_from_options
```

Reply:
Fixed